apps/public-docsite-v9/public/style-resolution.excalidraw

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
I tried the [excalidraw react component](https://www.npmjs.com/package/@excalidraw/excalidraw) but it's quite heavy and I'm not sure if we want to include it into our docs.... open to other opinions here.

I also considered [mermaid graph syntax](https://mermaid-js.github.io/mermaid/#/) but it's very constrained to creating flowcharts and graphs, and lacks the flexibility to illustrate this diagram well.